src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/shared/Quantity.java

```diff
@@ -10,9 +10,16 @@
 public class Quantity {
     public static final String MESSAGE_CONSTRAINTS =
             "Quantity should only contain numbers, it should be positive "
-                    + "and the largest acceptable quantity is 2147483647.";
+                    + "and the largest acceptable quantity is 40000.";
     private final int quantity;
 
+    /**
+     * Constructs a {@code Quantity} with a default value of 1.
+     */
+    public Quantity() {
+        quantity = 1;
+    }
```

Comment:
The default quantity is pertaining to order quantity, not the quantity model. This is assuming that all quantities default is 1. Is that true? Doesn't it make more sense to set the default quantity in `OrderDescriptor.java`?

```
private Quantity quantity = 1;
```